api/admin/service.go

```diff
@@ -128,6 +128,29 @@ func (service *Admin) AliasChain(_ *http.Request, args *AliasChainArgs, reply *a
 	return service.httpServer.AddAliasesWithReadLock("bc/"+chainID.String(), "bc/"+args.Alias)
 }
 
+// GetAliasesOfChainArgs are the arguments for calling GetAliasesOfChain
+type GetAliasesOfChainArgs struct {
+	Chain string `json:"chain"`
+}
+
+// GetAliasesOfChainArgs are the arguments for calling GetAliasesOfChain
+type GetAliasesOfChainReply struct {
+	Aliases []string `json:"aliases"`
+}
+
+// GetAliasesOfChain returns the all aliases of a chain
+func (service *Admin) GetAliasesOfChain(_ *http.Request, args *GetAliasesOfChainArgs, reply *GetAliasesOfChainReply) error {
```

Comment:
Can you please rename to `GetChainAliases` ? (Similar for args, responses, etc.)